python/pyspark/sql/tests.py

```diff
@@ -3036,6 +3052,9 @@ def test_toPandas_arrow_toggle(self):
         pdf = df.toPandas()
         self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
         pdf_arrow = df.toPandas()
+        # need to remove timezone for comparison
+        pdf_arrow["7_timestamp_t"] = \
+            pdf_arrow["7_timestamp_t"].apply(lambda ts: ts.tz_localize(None))
```

Comment:
@gatorsmile, can you be a bit explicit about the behavior we want? As we discovered that the current behavior of `df.toPandas()` is wrong and inconsistent with Spark SQL (because `df,toPandas()` doesn't respect `LOCAL_SESSION_TIMEZONE`), do you think we should:
1. be consistent with default `df.toPandas()` even though it is wrong
or:
2. be consistent with Spark SQL

Also this is not huge issue IMHO because we should fix the behavior of `df.toPandas()` soon anyway.